cm_sdk/models/alert_config/alert_config.py

```diff
@@ -0,0 +1,84 @@
+from cm_sdk.models import CloudManagerBase, CloudManagerEnum
```

Comment:
The file divisions correspond the the objects in the public API.  Some child objects have the same name but different semantics across packages.